operators/deploy/tenant-operator/values.yaml

```diff
@@ -17,6 +17,10 @@ configurations:
     url: "https://nextcloud.crownlabs.example.com/"
     user: username
     pass: password
+  userPvcSizeBytes: 1Gi
+  storageClassName: rook-nfs
+  rookCephNamespace: rook-ceph
+  mydrivePvcsNamespace: mydrive-pvcs
```

Comment:
Same here about starting these variables with mydrivePVCs.

Reply:
For this and the other comments, do you want `mydrivePVCs...` or `mydrivePvcs...`?

Reply:
To me mydrivePVCs... is slightly better, but that's not a big deal.